test_runner/regress/test_rest_broker.py

```diff
@@ -0,0 +1,135 @@
+import json
+import signal
+import time
+
+import requests
+from jwcrypto import jwt
+
+
+def test_rest_broker_happy(
```

Comment:
If I understand correctly, this test requires that the proxy is built with the `rest_broker` feature, or it fails. That's not cool, we should skip the test when not built with `rest_broker` instead